manifests/backup/mysqlbackup.pp

```diff
@@ -32,6 +32,11 @@
   $compression_command      = undef,
   $compression_extension    = undef,
 ) inherits mysql::params {
+  if $backuppassword =~ Sensitive {
+    $_backuppassword = $backuppassword.unwrap
```

Comment:
I know this isn't a big deal but would it be possible to rename this variable? I found when reviewing this PR with a colleague it took us a while to work out what it is doing. I would suggest using something such as `plaintext_backuppassword`. (this name change will need revisited throughout your codebase changes).

Reply:
Done 👍

Reply:
Now it's done.  I needed a second `push`.

Reply:
https://github.com/cocker-cc/puppetlabs-mysql/compare/f9d507f..8413dbe